Changelog.md

```diff
@@ -13,6 +13,7 @@ Bugfixes:
  * SMTChecker: Fix internal error on array implicit conversion.
  * SMTChecker: Fix internal error on fixed bytes index access.
  * References Resolver: Fix internal bug when using constructor for library.
+ * Yul Optimizer: LoopInvariantCodeMotion can move reading operations outside for-loops as long as the affected area is not modified inside the loop.
```

Comment:
I would consider this a feature.

Reply:
Oops. Moved to the wrong place. Fixed now.